metadata-ingestion/src/datahub/ingestion/source/file.py

```diff
@@ -273,15 +273,15 @@ def _iterate_file(self, file_status: FileStatus) -> Iterable[Tuple[int, Any]]:
     def iterate_mce_file(self, path: str) -> Iterator[MetadataChangeEvent]:
         schema = get_path_schema(path)
         fs_class = fs_registry.get(schema)
-        fs = fs_class.create_fs()
+        fs = fs_class.create()
```

Comment:
we'll probably need to add a mechanism for passing config here from the recipe, but we can leave that for a follow up PR